optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/roster/view/AbstractRosterView.java

```diff
@@ -84,7 +81,7 @@ public void setEmployeeList(List<Employee> employeeList) {
         this.employeeList = employeeList;
     }
 
-    @JsonSerialize(using = ScoreJacksonJsonSerializer.class)
+    @JsonSerialize(using = HardSoftScoreJacksonJsonSerializer.class)
```

Comment:
+1

But if you ObjectMapper.register the new OptaPlannerJacksonModule.createModule() (see 7.7.0 optaplanner docs), you can get rid of these entirely.
Do check if the client still works then.